cmd/ocm/create/cluster/cmd.go

```diff
@@ -61,6 +63,7 @@ var args struct {
 	computeMachineType string
 	computeNodes       int
 	autoscaling        c.Autoscaling
+	byovpc c.BYOVPC
```

Comment:
indentation issue

Reply:
byoVPC?

Reply:
must be exported like other cluster fields